client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -135,7 +135,7 @@
             <div
               v-tooltip="internId"
               class="o-hellip--nowrap text--right"
-              dir="rtl"
```

Comment:
why do we need that at all?
May it even work if we drop the prop?

Reply:
Turned out that dropping the attribute produces the same result hence done here https://github.com/demos-europe/demosplan-core/pull/1351/commits/9e0456ab0a8d0ee808b55725c3b5903ac14e79e1